<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.944, y2:0.943, stop:0 rgba(107, 117, 255, 255), stop:1 rgba(87, 255, 255, 255));}
QWidget&gt;QPushButton{
    background-color:  rgb(43, 43, 43);
	width: 150px;
    border-style: outset;
    border-width: 2px;
    border-radius: 8px;
    border-color: beige;
    font: bold 14px;
	color: rgb(85, 255, 255);
}
QWidget&gt;QPushButton:pressed{
    background-color: black;
   
}
QWidget&gt;QPushButton:hover{
    
	border-color: rgb(0, 0, 127);
   
}
QWidget&gt;QTableWidget{
  color: rgb(222, 105, 89);
 background: rgb(43, 43, 43);
font:bold 14px;

  
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QTableWidget" name="tabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>581</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <column>
        <property name="text">
         <string>Marca</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Model</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Culoare</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>An</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nr loc</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Capacitate </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nr inm</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Putere</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Combustibil</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nr KM</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data expirare RCA</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data expirare ITP</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Stergere</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Save data</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">	height:20px;
	width:120px;</string>
       </property>
       <property name="text">
        <string>Adaugare</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>481</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font: bold 28pt ;</string>
       </property>
       <property name="text">
        <string>CAR  MANAGEMENT  APP</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>31</width>
         <height>391</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>350</y>
         <width>176</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Editare tabel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>350</y>
         <width>176</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Save changes</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Cautare</string>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>-10</y>
         <width>16</width>
         <height>411</height>
        </rect>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>40</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: bold 22pt ;</string>
       </property>
       <property name="text">
        <string>Notificari</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>90</y>
         <width>201</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(42, 42, 42);
  color: rgb(222, 105, 89);
font: bold 14px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>350</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Hide</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>0</y>
         <width>16</width>
         <height>391</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
